types/marshal/marshal-tests.ts

```diff
@@ -0,0 +1,7 @@
+import Marshal from 'marshal';
```

Comment:
```suggestion
import Marshal = require('marshal');
```
https://github.com/clayzermk1/node-marshal#basic-use